Tiltfile

```diff
@@ -1,66 +1,23 @@
-# This loads a helper function that isn't part of core Tilt that simplifies restarting the process in the container
-# when files changes.
-load('ext://restart_process', 'docker_build_with_restart')
+if not os.path.exists('../tilt-support'):
```

Comment:
I'm very leery about going outside the repo.

Reply:
This is for local development and iteration. It's pretty standard.

Reply:
This doesn't clone the tilt-support repo. That's something you have to do manually. It's definitely not forcing arbitrary code from the internet on you 😄 

An alternative is to consider `tilt-support` a Tilt extension:

```
v1alpha1.extension_repo(name='tilt-support', url='https://github.com/operator-framework/tilt-support')
v1alpha1.extension(name='tilt-support', repo_name='tilt-support', repo_path='.')
load('ext://tilt-support', 'deploy_repo')
```

Both would load the same code

Reply:
I just said I'm "leery". I haven't seen many (if any?) projects that look outside their repository unless explicitly configured to do so by the user (e.g. `--with-software=/path/to/software`). Then again, I'm also leery about pulling random software versions from the 'net (given my corporate security background).
That said, if it's just looking for something, then it's not so bad.

Reply:
Ok, cool. Yeah this is by far the easiest way.